apps/example/webapps/showcase/previews/ack-button.pug

```diff
@@ -1,5 +1,5 @@
-panel(type="info" title="ack-button")
-    .content
+.col-md-12
```

Comment:
[Other branch](https://github.com/ozanmuyes/scada.js/tree/paged-showcase) will conclude this request.